src/client/components/AddMeal.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+
+export default function AddMeal() {
+  const [inputState, setInputstate] = useState(false);
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [location, setLocation] = useState("");
+  const [When, setWhen] = useState("");
```

Comment:
Nitpick(naming): Prefer camelCase for variables and functions, save PascalCase for classes and types.